src/Microsoft.SqlTools.ServiceLayer/SqlProjects/SqlProjectsService.cs

```diff
@@ -44,6 +44,8 @@ public void InitializeService(ServiceHost serviceHost)
             serviceHost.SetRequestHandler(OpenSqlProjectRequest.Type, HandleOpenSqlProjectRequest, isParallelProcessingSupported: true);
             serviceHost.SetRequestHandler(CloseSqlProjectRequest.Type, HandleCloseSqlProjectRequest, isParallelProcessingSupported: true);
             serviceHost.SetRequestHandler(NewSqlProjectRequest.Type, HandleNewSqlProjectRequest, isParallelProcessingSupported: true);
+            serviceHost.SetRequestHandler(GetCrossPlatformCompatiblityRequest.Type, HandleGetCrossPlatformCompatibilityRequest, isParallelProcessingSupported: true);
+            serviceHost.SetRequestHandler(UpdateProjectForCrossPlatformRequest.Type, HandleUpdateProjectForCrossPlatformRequest, isParallelProcessingSupported: false);
```

Comment:
`isParallelProcessingSupported: false` means this method will run synchronously on the message dispatcher thread.  i.e. it should run quickly and not access a potentially blocking resource like a DB or network.  Or it should handle it's own task management.  Just double-checking that's intended.

Reply:
It should run quickly.  This edits the project file on disk, and could potentially cause issues if multiple operations try to make simultaneous/conflicting edits.  Is that a reasonable usage?  I could also make all of these parallelizable, and use something within SqlProjectService to handling locking on a per-project basis.